packages/esm-patient-notes-app/src/notes/notes.component.tsx

```diff
@@ -16,7 +16,11 @@ export default function Notes({ basePath, patient, patientUuid, showAddNote }: N
     <NotesContext.Provider value={{ patient, patientUuid }}>
       <BrowserRouter basename={`${window.spaBase}${basePath}/encounters/notes`}>
         <Switch>
-          <Route exact path="/" render={() => <NotesDetailedSummary showAddNote={showAddNote} />} />
+          <Route
+            exact
+            path="/"
+            render={() => <NotesDetailedSummary patientUuid={patientUuid} showAddNote={showAddNote} />}
```

Comment:
Hm not sure why we need to transport `patientUuid`. It can be received from the `NotesContext`. Something I missed?

Reply:
Yeah, no reason for this. I must've missed it somehow.